Apps/Examples/Examples/SwiftUI Examples/Util/Constants.swift

```diff
@@ -20,5 +20,6 @@ extension CameraBoundsOptions {
 }
 
 extension StyleURI {
+    static let streetsV12 = StyleURI(rawValue: "mapbox://styles/mapbox/streets-v12")!
```

Comment:
Is it time to present v12 streets style?

Reply:
I don't think we should include this in this PR, we have a PR https://github.com/mapbox/mapbox-maps-ios/pull/1725 to update core styles but it's currently blocked.